xmodule/item_bank_block.py

```diff
@@ -155,9 +161,9 @@ def make_selection(cls, selected, children, max_count):
 
         return {
             'selected': selected,
-            'invalid': invalid_block_keys,
-            'overlimit': overlimit_block_keys,
-            'added': added_block_keys,
+            'invalid': sorted(invalid_block_keys),
+            'overlimit': sorted(overlimit_block_keys),
+            'added': sorted(added_block_keys),
```

Comment:
Why are these now sorted? Is it to facilitate testing or fix a bug? I would assume it's better to have `added` reflect the order in which the blocks are added, but I don't think it matters.

Reply:
To facilitate testing